doc/environment.adoc

```diff
@@ -0,0 +1,247 @@
+= Lua Environment
```

Comment:
Should have author and revision.

Reply:
I disagree. This is stored in the Git history.

Reply:
i also disagree

Reply:
Let's keep any author metadata to a macro that we (eventually) import, it should say something like "Minetest Docs Team and contributors." for the author globally if at all.